internal/store/files/WorldGeosForPlaceRecognition.csv

```diff
@@ -17,15 +17,15 @@ country/ASM,Country,American Samoa,"Earth,country/USA,northamerica,oceania",4503
 country/ATF,Country,French Southern Territories,"Earth,africa,antarctica",196
 country/ATG,Country,Antigua and Barbuda,"Earth,northamerica",93219
 country/AUS,Country,Australia,"Earth,oceania",25688079
-country/AUT,Country,Austria,"Earth,europe",8955797
+country/AUT,Country,Austria,"Earth,europe,EuropeanUnion",8955797
```

Comment:
For now, can you drop these containment lines?

Reply:
done